packages/asset-uploader/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+// Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
+{
+  "extends": "../../node_modules/tsconfig/base.json",
+  "include": ["./src/**/*.ts", "./src/**/*.tsx"],
+  "compilerOptions": {
+    "lib": ["DOM", "DOM.Iterable", "ES2019"],
+    "module": "commonjs",
+    "target": "ES2019",
+    "strict": true,
+    "baseUrl": ".",
+    "paths": {
+      "~/*": ["./src/*"]
+    },
+    "outDir": "./lib"
+  }
+}
```

Comment:
you don't need everything that has to do with browser, because that's package for node only

Reply:
we probably need a default tsconfig/node.json  for all pure node packages

Reply:
Updated it but doesn't fix anything :/